login/service_blackbox_test.go

```diff
@@ -543,10 +543,11 @@ func (s *serviceBlackBoxTest) TestUnapprovedUserLoginUnauthorized() {
 	}
 
 	err = s.loginService.Perform(authorizeCtx, dummyOauth, s.Configuration)
-	require.NotNil(s.T(), err)
+	require.Nil(s.T(), err)
```

Comment:
This changes the test case, right?

Reply:
seems wrong to me. It is passing cause of dummyOauth. Will change this.